### PR TITLE
Deploying private apps section update

### PR DESCRIPTION
**Pull request by jonathanglassman**
What
----

Updated the deploying private apps section in line with the doc-a-thon feedback.

How to review
-------------

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).
1. Check that it fulfils story https://www.pivotaltracker.com/story/show/167640865

Who can review
--------------

Anyone except Jon Glassman or Andy Hunt

### CODE REVIEW

File: source/documentation/deploying_apps/index.erb [hunk 1]
```diff
@@ -74,23 +74,27 @@ A possible exception to this is if your org is mature and has pre-existing space
...
 
-You can deploy apps so they are not publicly accessible to everyone through the internet.
+By default, all apps you deploy on Cloud Foundry are [publicly accessible](deploying_apps.html#deploying-public-apps) to everyone through the internet.
+
+If you need to restrict access to a public app, for example to whitelist IPs, you should refer to the documentation on [route services](/deploying_services/route_services/#route-services).
+
+If you do not want your apps to be publicly accessible at all, you must deploy your apps on the `apps.internal` domain. Two common use cases for this are that you want to:
+
+- deploy a task scheduling app on the GOV.UK PaaS that does not need to be publicly accessible
+- test a beta version of an app in your production environment that only your team can access
```
Comment (AP-Hunt) [inline source/documentation/deploying_apps/index.erb]: Private apps aren't accessible from the outside, without respect for who the user is. In this case, the thing that can access the beta version is other apps with configured routes to it.

I think in hindsight, there's probably a better example than this. I'll ask in Slack.

Comment (AP-Hunt) [reply source/documentation/deploying_apps/index.erb]: @jonathanglassman From Friday's discussion, it looks like the most common usage of private apps is for deploying micro-service (the technical term) architected services (the citizen-facing term).

Comment (jonathanglassman) [reply source/documentation/deploying_apps/index.erb]: @AP-Hunt thanks. So what are architected services?

**Comment by jonathanglassman** [comment]
Changes do not affect user behaviour therefore 2i not required. 

As @tlwr has reviewed and ok'ed, merging and moving to approval.

**Pull request merged**